### ISSUE TITLE
Extend support for common SET commands from sql clients

### ISSUE THREAD

**Issue by guidoiaquinti** [youtube/vitess#3111]
Opening this issue to keep track of an enhancement for #3056 by @bbeaudreault. As discussed already in Slack, `SET` operations are not correctly parsed if we use `'` or `"`:

```
mysql> SET CHARACTER SET utf8mb4;
Query OK, 0 rows affected (0.00 sec)

mysql> SET CHARACTER SET "utf8mb4";
ERROR 1105 (HY000): vtgate: http://vitess:15001/: syntax error at position 28 near 'utf8mb4'

mysql> SET CHARACTER SET 'utf8mb4';
ERROR 1105 (HY000): vtgate: http://vitess:15001/: syntax error at position 28 near 'utf8mb4'
```

It could be nice to add as well the support for `SET lc_messages`

**Comment by bbeaudreault** [comment]
The parse error will be fixed in https://github.com/youtube/vitess/pull/3110.

`SET lc_messages` would be harder. I recommend configuring your mysql server at startup instead, because Vitess can't change session variables due to connection pooling.

**Comment by giaquinti** [comment]
We already set the variable server side but there are unfortunately few apps (like phpMyAdmin) that are trying to override it when they establish a new session. What do you think about adding it to the ignore list?

> `lc_messages` can be set at server startup and has global and session values that can be modified at runtime. Thus, the error message language can be changed while the server is running, and individual clients each can have a different error message language by changing their session `lc_messages` value to a different locale name.
>
> ref: https://dev.mysql.com/doc/refman/5.7/en/error-message-language.html

**Comment by bbeaudreault** [comment]
Ok, I would not be opposed to adding it to the ignore list. I did see that doc, and I don't think we can support the session value. So ignore list could work, if that works for you.

**Comment by giaquinti** [comment]
Awesome, thanks! 🚀

**Comment by guidoiaquinti** [comment]
Fixed by #3110
